packages/components/src/InputDate/InputDate.tsx

```diff
@@ -40,10 +41,17 @@ interface InputDateProps
 }
 
 export function InputDate(inputProps: InputDateProps) {
+  const formFieldActionsRef = useRef<FieldActionsRef>(null);
   return (
     <DatePicker
       selected={inputProps.value}
-      onChange={inputProps.onChange}
+      onChange={val => {
+        inputProps?.onChange(val);
+        formFieldActionsRef.current?.setValue(
+          // Text Input has a string type
+          `${val.getMonth()}/${val.getDay()}/${val.getFullYear()}`,
+        );
```

Comment:
instead of manually parsing the date, since the format of it is highly dependent on the system's settings, you can move this bit down to the activator

<img width="662" alt="Cursor" src="https://user-images.githubusercontent.com/15986172/230434475-5bd737ed-e6d5-4d04-8d97-a8a526af74a0.png">

Fixes the issue and we won't have to deal with date formatting

Reply:
Nice!! Ok, on it.